integration-test/background/atb.js

```diff
@@ -137,27 +146,35 @@ describe('search workflow', () => {
         await wait.forSetting(bgPage, 'extiSent')
 
         // grab current atb data
-        const data = await request('https://duckduckgo.com/atb.js', { json: true })
+        let data = await fetch('https://duckduckgo.com/atb.js')
+        data = await data.json()
         todaysAtb = data.version
         lastWeeksAtb = `v${data.majorVersion - 1}-${data.minorVersion}`
         twoWeeksAgoAtb = `v${data.majorVersion - 2}-${data.minorVersion}`
     })
     afterAll(async () => {
-        await harness.teardown(browser)
+        try {
+            await harness.teardown(browser)
+        } catch (e) {}
     })
     beforeEach(async () => {
-        await bgPage.evaluate((atb) => dbg.settings.updateSetting('atb', atb), twoWeeksAgoAtb)
+        try {
+            await bgPage.evaluate((atb) => dbg.settings.updateSetting('atb', atb), twoWeeksAgoAtb)
+        } catch (e) {}
     })
     it('should not update set_atb if a repeat search is made on the same day', async () => {
         // set set_atb to today's version
         await bgPage.evaluate((todaysAtb) => dbg.settings.updateSetting('set_atb', todaysAtb), todaysAtb)
 
         // run a search
         const searchPage = await browser.newPage()
-        searchPage.goto('https://duckduckgo.com/?q=test')
-
-        await bgPage.waitForResponse(res => res.url().match(/atb\.js/))
```

Comment:
this part was not moved over - not 100% sure if we need it, but I guess it's checking if atb.js has loaded?

Reply:
Maybe there was an intended test in there, but it always times out and can cause errors (I don't think this is the right way to test whether a sub-request was made, but I could be wrong). The rest of the test succeeds without it. 

If we want to check that the atb.js file loaded, we could probably add a separate test for that?